src/apps/safety/views/Home/index.jsx

```diff
@@ -28,9 +26,9 @@ const Home = () => {
          <StyledCardList>
             <DashboardTile
                title={t(address.concat('safety checks'))}
-               count={count}
+               count={count || '...'}
```

Comment:
```suggestion
               count={count}
```

Not needed because it will result in `'...'` if the count is zero which it is for a default value